lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -873,6 +898,9 @@ instance ToSchema (PostTransactionData t) where
 instance ToSchema (PostTransactionFeeData t) where
     declareNamedSchema _ = declareSchemaForDefinition "ApiPostTransactionFeeData"
 
+instance ToSchema PostExternalTransactionData where
+    declareNamedSchema _ = declareSchemaForDefinition "ApiPostExternalTransactionData"
```

Comment:
```suggestion
    declareNamedSchema _ =
        declareSchemaForDefinition "ApiPostExternalTransactionData"
```

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Reply:
done